src/arkreactor/VM/Scope.cpp

```diff
@@ -2,22 +2,19 @@
 
 #include <limits>
 
-#define push_pair(id, val) m_data.emplace_back(std::pair<uint16_t, Value>(id, val))
-#define insert_pair(place, id, val) m_data.insert(place, std::pair<uint16_t, Value>(id, val))
-
 namespace Ark::internal
 {
     Scope::Scope() noexcept
     {}
 
     void Scope::push_back(uint16_t id, Value&& val) noexcept
     {
-        push_pair(std::move(id), std::move(val));
+        m_data.emplace_back(std::pair<uint16_t, Value>(std::move(id), std::move(val)));
```

Comment:
Don't need `std::pair` here.
The emplace_back handles that for you.

Just use: `m_data.emplace_back(std::move(id), std::move(val));`